### PR TITLE
i18n(es): Updated `getting-started.mdx`

### PR DESCRIPTION
**Pull request by Waxer59**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
Updated `getting-started.mdx`
* [`7bed5a2`](https://github.com/withastro/docs/commit/7bed5a234ccead2f796f83b09b523e5e4ffc19aa#diff-c7de13e86d91f5c1b553e49684e8e7d8de76486ab569f8dc5c88805812257e0f)

### CODE REVIEW

File: src/content/docs/es/getting-started.mdx [hunk 1]
```diff
@@ -99,7 +100,12 @@ Nuestra [guía de instalación](/es/install/auto/) tiene instrucciones completas
 
 [Historial de cambios de Astro](https://github.com/withastro/astro/blob/main/packages/astro/CHANGELOG.md)
 
+<TranslatorList githubRepo="withastro/docs">
+{/* Turn this HTML heading into a Markdown h2 — ## — when translating this page. */}
+<h2>Traducir</h2>
```
Comment (Yan-Thomas) [inline src/content/docs/es/getting-started.mdx:105]: Here the heading should become a Markdown h2!
```suggestion
## Traducir
```

**Comment by Waxer59** [comment]
I hadn't noticed 😅, thanks for the review! 🤠

**Review by dreyfus92 — APPROVED** [review]
LGTM ✅

**Review by Yan-Thomas — APPROVED** [review]
LGTM!

**Pull request merged**